apps/etl/management/commands/extract_glide_data.py

```diff
@@ -1,10 +1,36 @@
 from django.core.management.base import BaseCommand
 
 from apps.etl.etl_tasks.glide import ext_and_transform_glide_historical_data
+from apps.etl.management.commands.extract_usgs_data import validate_date_format
```

Comment:
If this `validate_date_format` is a utility function, it should be moved appropriately. Not a good to import from extract_usgs_data for glide stuffs.